app/forms/check_your_answers_form.rb

```diff
@@ -71,4 +72,20 @@ def displayable_address(address)
      address.postcode,
      address.country].reject
   end
+
+  def should_be_renewed
+    no_invalid_business_type_change? && same_company_no?
+  end
+
+  def no_invalid_business_type_change?
```

Comment:
Just a personal preference (so tell me to sod off) but I believe dropping the `no_` makes the intent clearer. So just `invalid_business_type_change?`. 🤔

Reply:
I agree, it is awkward! Will try to rephrase this.